packages/btc/tests/shared/utils.ts

```diff
@@ -109,11 +109,47 @@ export function createAccount(props: { privateKey: string; network?: bitcoin.Net
 }
 
 /**
- * Sign and broadcast a transaction to the service
+ * Sign a PSBT with one or multiple BtcAccounts
+ */
+export function signPsbt(props: {
+  psbt: bitcoin.Psbt;
+  account: Account | Account[];
```

Comment:
I think `Account[]` is enough

Reply:
This is for code compatibility, most transactions in the tests only require one account to sign. I'll refactor the tests and break the structure of Account later in another PR, so maybe this function can also be refactored then.